bhyve-api/src/structs.rs

```diff
@@ -319,6 +319,14 @@ pub struct vm_run_state {
     pub _pad: [u8; 3],
 }
 
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct vm_dirty_tracker {
+    pub start_gpa: u64,
+    pub len: size_t,
+    pub pfns: *mut c_void,
```

Comment:
These should probably reflect the naming in the OS header, with the `vdt_` prefixes.

Reply:
Interesting; this isn't quite universal (e.g., `cpuid` in `vm_run_state` is `vcpuid` in the kernel). On reflection, it appears that `vmm_dirty_tracker` may be misnamed on several levels; perhaps it should be named `vm_dirty_tracker` and the `vdt_` prefixes removed; that would be more consistent with other Bhyve data structures. What do you think? I already need to fix the comment in the kernel.

Reply:
(Talked about this on Matrix; we'll stick with what we have and the definitions here have been changed to match the kernel.)